pandas/_libs/tslibs/offsets.pyx

```diff
@@ -4696,13 +4696,9 @@ _lite_rule_alias = {
     "BYS": "BYS-JAN",  # BYearBegin(month=1),
 
     "Min": "min",
-    "min": "min",
-    "ms": "ms",
-    "us": "us",
-    "ns": "ns",
 }
 
-_dont_uppercase = {"h", "bh", "cbh", "MS", "ms", "s"}
+_dont_uppercase = {"min", "h", "bh", "cbh", "s", "ms", "us", "ns"}
```

Comment:
It never made much sense to me that `'MS'` was in this list. It's been there since forever, but it looks odd - `'MS'` is already uppercase 😄

Reply:
agree, it was always confusing for me.